playbooks/amc.yml

```diff
@@ -30,9 +30,9 @@
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    # Do we need need edxapp 2 times here????
```

Comment:
It's also there in the edx upstream repo so I imagine it must serve some purpose.